dotcom-rendering/src/web/components/ArticlePage.tsx

```diff
@@ -34,61 +35,70 @@ type Props = {
 export const ArticlePage = ({ CAPIArticle, NAV, format }: Props) => {
 	return (
 		<StrictMode>
-			<Global
-				styles={css`
-					/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
-					/* The not(.src...) selector is to work with Source's FocusStyleManager. */
-					*:focus {
-						${focusHalo}
-					}
-					::selection {
-						background: ${brandAlt[400]};
-						color: ${neutral[7]};
-					}
-				`}
-			/>
-			<SkipTo id="maincontent" label="Skip to main content" />
-			<SkipTo id="navigation" label="Skip to navigation" />
-			{(format.design === ArticleDesign.LiveBlog ||
-				format.design === ArticleDesign.DeadBlog) && (
-				<SkipTo id={'key-events-carousel'} label="Skip to key events" />
-			)}
-			<Island clientOnly={true} deferUntil="idle">
-				<AlreadyVisited />
-			</Island>
-			<Island clientOnly={true} deferUntil="idle">
-				<FocusStyles />
-			</Island>
-			<Island clientOnly={true} deferUntil="idle">
-				<Metrics
-					commercialMetricsEnabled={
-						!!CAPIArticle.config.switches.commercialMetrics
-					}
+			<ServerSideTestProvider tests={CAPIArticle.config.abTests}>
```

Comment:
the only significant change in this file should be adding this `ServerSideTestProvider` wrapper.